Automattic/jetpack
# Blaze: update wording for the Post list link and editor panel

Fixes #28078
Fixes #28079

#### Changes proposed in this Pull Request:

- Update wording for the Post list link
- Avoid checking in WordPress install added when installing
- Update link anchor text as well
- Update icon and wording for post-publish editor panel.

<img width="276" alt="Screenshot 2023-01-03 at 18 29 11" src="https://user-images.githubusercontent.com/426388/210412242-22186875-b3cd-47cd-8009-6c6320ee7b5f.png">

<img width="740" alt="image" src="https://user-images.githubusercontent.com/426388/210412388-cb18e12e-d386-440c-8195-85fc06bf27f8.png">

#### Other information:


#### Jetpack product discussion

* See links in #28079

#### Does this pull request change what data or activity we track or use?

* No

#### Testing instructions:

* On a WoA site, switch to this branch (using the Jetpack Beta plugin for example)
* Go to Posts 
* See the updated post list
* Go to Posts > Add New
* Write a new post and publish it.
* See the upated panel.
